### PR TITLE
Feat : Added product list page

### PR DESCRIPTION
**Pull request by sushil-mod**
Added : product list page with filters

### CODE REVIEW

File: screens/html/product-list.html [hunk 1]
```diff
@@ -0,0 +1,302 @@
...
+    <link
+    rel="stylesheet"
+    href="https://pro.fontawesome.com/releases/v5.10.0/css/all.css"
+    integrity="sha384-AYmEC3Yw5cVb3ZcuHtOA93w35dYTsvhLPVnYs9eStHfGJvOvKxVfELGroGkvsg+p"
+    crossorigin="anonymous"
+  />
+    <link rel="stylesheet" href="https://dreamy-wright-6bb3af.netlify.app/components/components.css">
+
+    <link rel="stylesheet" href="/style.css">
+    <title>Document</title>
```
Comment (t007rushi) [inline screens/html/product-list.html]: change title name

**Review by SuyashTiwariCoder — APPROVED** [review]
Looks good to me , you can merge .

File: screens/html/product-list.html [hunk 2]
```diff
@@ -0,0 +1,302 @@
...
+                                <label for="two-star">Collectables </label>
+                            </li>
+                            <li>
+                                <input type="checkbox"  name="catagorie" >
+                                <label for="one-star"> Merchandise</label>
+                            </li>
+                        </ul>
+                    </div>
+                    <p class="filter-head padd-md" >Price</p>
+                <input class="filter-slider" type="range" min="1" max="100" value="50">
```
Comment (t007rushi) [inline screens/html/product-list.html:100]: show this range on UI

File: screens/html/product-list.html [hunk 3]
```diff
@@ -0,0 +1,302 @@
...
+
+                        <span class=" card-rating">4.2 <i class="fas fa-star"></i>(224)</span>
+
+                        <span>Price :<h3 class="inline-block">₹ 2,000</h3> </span> 
+
+                        </div>
+                        
+
+
+                        <button class="btn btn-shop card-fix-btn card-btn-sec " > <i class="fas fa-shopping-cart"></i> Add to cart</button>
```
Comment (t007rushi) [inline screens/html/product-list.html:192]: have some spacing between icon & text

File: screens/html/product-list.html [hunk 4]
```diff
@@ -0,0 +1,302 @@
...
+            <div class="navbar-links flex-center">
+              <a class="nav-login-link" href="./screens/html/login.html">
+                <button class="nav-login-btn">Login</button>
+              </a>
+              <ul class="nav-list">
+                <li class="nav-list-item">
+                  <a class="nav-list-link" href="#"><i class="far fa-heart nav-link-icon"></i></a>
+                </li>
+                <li class="nav-list-item">
+                  <a class="nav-list-link icon-badge pos-rel " href="#"><i class="fas fa-cart-plus nav-link-icon"></i><div class="badge-number flex-center badge-on-icon pos-abt">4</div></a>
```
Comment (t007rushi) [inline screens/html/product-list.html]: badge icon can be primary or cta color just suggestion as blue contrasts whole styling

**Review by t007rushi — APPROVED** [review]
Have some spacing in card contents else with minor changes looks good

File: screens/html/product-list.html (hunk 1 above)
Comment (sushil-mod) [reply screens/html/product-list.html]: done

File: screens/html/product-list.html (hunk 2 above)
Comment (sushil-mod) [reply screens/html/product-list.html:100]: done thanks for pointing out ,i forgot

File: screens/html/product-list.html (hunk 4 above)
Comment (sushil-mod) [reply screens/html/product-list.html]: done

File: screens/html/product-list.html (hunk 3 above)
Comment (sushil-mod) [reply screens/html/product-list.html:192]: yeah sure

**Pull request merged**